Various/tompad_obliques/obliques.lua

```diff
@@ -0,0 +1,215 @@
+-- @noindex
```

Comment:
Loading the random messages from this obliques.lua will likely choose the first two metadata lines sometimes (`@noindex` is required due to the .lua file extension–a file without an extension would not need any metadata).

Reply:
Ok. But what resource type should I select in Upload Tools for that file? I had a .txt file and tested different types but the path to .txt changed to a one I didn't want. Can I edit file here online or will I need to do a new upload from Upload Tools?

Reply:
The resource type should remain the same (ReaScript) to have the file installed into the Scripts folder. The file extension can be set in "storage file name" independently of the resource type (I suggest not using any extension for that file).

You can edit the files right here from this pull request's page.

Reply:
Ok, I made a change - what happens now? (Yes I'm a noob on GitHub;-))

Reply:
And another question......why dont my 2 files live in one folder? Now it looks like my obliques-file is in a folder but the script is not in same folder...

Reply:
Now the script can safely be released (I'll merge this tomorrow, in case you want to move the obliques file before the script gets released).

The upload tool automatically puts additional files into a subdirectory to keep the repository tidy when scripts consist of many files. (This can be bypassed by prefixing the storage file name with `../`.)

Reply:
Ok, so if I am satisfied with having obliques file in a subdirectory there is no more I need to do?
After some thinking I think its good to have it in subdirectory, as you say, keeps repository tidy.
Looking forward to see my script in ReaPack - thanks for all cfillion!